sections/error-handling.md

```diff
@@ -66,3 +66,37 @@ A sample 500 Internal Service error:
   }]
 }
 ```
+
+## Warning and Information Responses
+
+It may be appropriate in some scenarios for resource servers to return information about non-critical errors or other significant conditions.
+When employing extra-data messages, the intent of such messages **MUST** be clearly articulated in API specifications, **MUSTNOT** unduely constrain or direct client behaviour (i.e. avoid close coupling) and **MUST NOT** be stateful (i.e. do not represent retention of client state by the server). Warning and information messages **SHOULD** be returned only in exceptional circumstances, **MUST ONLY** be returned by a server and **MUST NOT** be passed to a server by a client.
```

Comment:
I'm not really sure that we can really control what a client does, but it seems kind of moot. If it's about passing warnings from a client to a server as a part of an API, again, I'm not sure what this achieves. If an API needs such a design, why would it be excluded? A common format for warnings makes sense in general though.